src/main/java/edu/cornell/kfs/kew/mail/service/impl/CuActionListEmailServiceImpl.java

```diff
@@ -0,0 +1,21 @@
+package edu.cornell.kfs.kew.mail.service.impl;
+
+import org.apache.commons.lang3.StringUtils;
+import org.kuali.kfs.core.api.config.property.ConfigContext;
+import org.kuali.kfs.kew.mail.service.impl.CustomizableActionListEmailServiceImpl;
+
+public class CuActionListEmailServiceImpl extends CustomizableActionListEmailServiceImpl {
+
+    @Override
+    protected boolean isProduction() {
+        return super.isProduction() || shouldForceProductionBehavior();
+    }
+
+    protected boolean shouldForceProductionBehavior() {
```

Comment:
Are we going to need to set the cu.actionlist.email.environment to prod in a non-prod env to get prod like behaviour? Why do we need prod like behavior in non-prod if this service is using the regular email service? Thank you.

Reply:
The two services behave a little differently for their Production modes:

* With the regular email service, it uses a KFS parameter to control whether to log the email, send to a test address, or send to the actual address. (Note that this parameter is ignored when running in Production.)
* With the action list email service and its related email content service, they rely on config properties to control whether the action list emails should be sent to the actual recipient or a test address, as well as whether to include extra debug-related content in the email body. (Note that the test address for this service uses a different parameter than the one used by the regular email service.)
  * The setup of the debug email content and workflow test address take place BEFORE the email is passed to the regular email service for processing.

This PR updates the setup for the action list and email content services, so that property-configured Production-style behavior can be enabled for both of them in a non-Production environment if necessary. That way, we can use a non-Production environment to test out what the email structure would look like in a Production-like setup.

Reply:
What values will we use for cu.actionlist.email.environment ?

Reply:
By default, this PR sets up the "cu.actionlist.email.environment" property so that it equals the value of the "environment" property. If we wanted to force this setting to the PROD value in certain environments, we could have DevOps override the property to explicitly equal "kfs-prod".

Reply:
I find the use of this property to force prod like behavior is potentially confusing. Since we are adding custom code to force production like behavior should we use a more explicit way? Would a system param be better? The system parameter description could also better explain what the "production like" behavior is.

Reply:
The old Cynergy code was using a config property to set up the Production-like behavior here. However, since we're changing the feature for KEW-to-KFS compatibility anyway, I agree that it makes more sense to control this setting via a parameter instead. I will work on implementing that change, and will also resolve the merge conflict that is being reported now on this PR.